### PR TITLE
nav bar

### LINKED ISSUE #5: create nav bar
create the nav bar

### LINKED ISSUE #6: make the nav bar reactive
add js to the nav bar

### PR DESCRIPTION
**Pull request by smaristeinar**
the base of the nav bar is ready
+responsive for web tablet and phone
+hamburger menue works
-need to look at the shadow elemet on the nav bar 
-needs size tweeking acording to the xd document 
-yellow line might need to be reconsidered

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -4,9 +4,29 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Blue Recycling</title>
 </head>
 <body>
+    <div id="navBar">
```
Comment (ljwedin) [inline index.htm]: Should we use the same case type throughout the project? You use camel case here and kebab on line 19.

Comment (smaristeinar) [reply index.htm]: absolutely

File: style.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    @media screen and (min-width: 1020px){
+        #burger{
+            visibility: hidden;
+        }
+        #nav-items {
+            visibility: visible;
+            padding-top: 0px;
+            display: flex;
+            justify-content: flex-end;
+            gap:66px;
```
Comment (ljwedin) [inline style.scss]: Add a blank space after gap: to follow formatting rules

**Review by ljwedin — CHANGES_REQUESTED** [review]
Great job!

File: index.htm (hunk 1 above)
Comment (LinusZachrisson) [reply index.htm]: Great job!

**Pull request merged**